sync:
  image: n3llyb0y/wait
  links:
    - auth
    - postgres
    - rabbitmq
    - smtp
  environment:
    PORTS: "9497 5432 5672 25"
    TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"
auth:
   image: wazopbx/wazo-auth-tests
   environment:
    - "TZ=UTC+1"
    - "XIVO_UUID=the-predefined-xivo-uuid"
   ports:
     - "9497"
   volumes:
     - "../../..:/usr/src/wazo-auth"
     - "./etc/wazo-auth/config.yml:/etc/wazo-auth/config.yml"
     # - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python2.7/site-packages/xivo"
     # - "${LOCAL_GIT_REPOS}/xivo-bus/xivo_bus:/usr/local/lib/python2.7/site-packages/xivo_bus"
   links:
     - "postgres"
     - "rabbitmq"
     - "smtp"
   command: "wazo-auth"
postgres:
  image: wazopbx/wazo-auth-db
  ports:
    - "5432"
rabbitmq:
  image: rabbitmq
  ports:
    - "5672"
smtp:
  image: python:2.7.13
  ports:
    - "25"
  command: "python -m smtpd -n -c DebuggingServer 0.0.0.0:25"
